fix(karma-webpack): disable karma watch; use webpack watch only

daKmoR opened the pull request:
This PR contains a:

- [x] **bugfix**

### Motivation / Use-Case

currently, karmas watch retriggers builds and we have a debounce to mitigate that.
However, when watching is wanted that doesn't work.

=> so disable karma watch mode
=> get rid of the debounce
=> use only webpacks watch

LarsDenBakker commented:
Even with a bigger timeout, it still runs the tests multiple times for me after a small change.

LarsDenBakker commented:
I think it's unrelated to the debounce, it looks like every so often it's spawning extra build processes and it just keeps growing.

daKmoR commented:
@LarsDenBakker so "ditching" karma watch all together lets us get rid of the full debounce and should also enable a better watch mode without duplicate builds.

can you pls check again :)

LarsDenBakker commented:
Yep, that fixes it 👍

daKmoR commented:
@matthieu-foucault I updated the merge request and cleaned up the history.

I think once this is merged we should have another alpha release (together with the other unreleased fix) - can you make it happen pls 🤗

rynclark commented:
I’d merge but I’m on mobile. Thanks for the PR, awesome work!

daKmoR commented:
friendly reminder for this pull and release request 🤗

matthieu-foucault commented:
Will take care of this later today

Le mar. 1 janv. 2019 à 07:53, Thomas Allmer <notifications@github.com> a
écrit :

> friendly reminder for this pull and release request 🤗
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/webpack-contrib/karma-webpack/pull/386#issuecomment-450739345>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ACj0dtbjMJjVX9UWb_DWXA9e8Njam6CLks5u-4RhgaJpZM4ZknC7>
> .
>
